system-tests/lib/infra/blockchain.go

```diff
@@ -0,0 +1,51 @@
+package infra
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/rs/zerolog"
+)
+
+func WaitForRPCEndpoint(lggr zerolog.Logger, url string, timeout time.Duration) error {
```

Comment:
Let's add TODO here, this function is here because we have unreliable ingresses in CRIB